### ISSUE TITLE
Double size

### ISSUE THREAD

**Issue by z-vr** [scottcorgan/content-length#1]
Hi mate, cool package, but I'm not sure if it works 100% correct (at least with Node 6), because `.write()` will be called automatically from `.end()`. I understand that you were trying to make it count bytes when `.end()` is called as well, but it does not seem necessary. 

https://github.com/nodejs/node/blob/master/lib/_http_outgoing.js#L773

**Comment by z-vr** [comment]
what I've done is 
```
const write = res.write.bind(res);
let byteLength = 0;
res.write = function writeResp(payload) {
    if (payload) byteLength += payload.length;
    write(...arguments);
};
```
and that was it

**Comment by scottcorgan** [comment]
Thanks for catching that 💯 . Fixed in 40d56385634856727731b131eb9426fac2da230a and published as `1.0.1`.
